### DISCUSSION TITLE
Pocketbase down No version satisfies ~0.10.0

### DISCUSSION THREAD

**Discussion by rubenvisser22** [pockethost/pockethost#220] [Q&A] [answered]
No version satisfies ~0.10.0

Thanks for your help!

**Comment by benallfree** [comment]
@rubenvisser22 What is the instance name?

**Reply by rubenvisser22** [reply]
https://forwardfuture.pockethost.io/_/?#/collections?collectionId=&filter=&sort=-created

Thanks for the fast reply!

**Reply by benallfree** [reply, marked as answer]
@rubenvisser22 Fixed! Nice catch.

**Reply by rubenvisser22** [reply]
Thanks you so much!
